example/priv/static/js/app.js

```diff
@@ -1,25 +1,86 @@
-var container = document.getElementById('routes-container');
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
This is not right. This is the "compiled" version, we want the original version here.

Reply:
oh that's right! We need to ignore those two folders (`example/priv/static/css` and `example/priv/static/js`) and remove the files from your commit, as they are compiled files.